_posts/2016-11-07-recurse-center-week-6.md

```diff
@@ -43,10 +43,10 @@ I had a lot of bottled up praise that I wanted to share. I had even considered p
 
 It gave me a sense of closure that I could share those thoughts with all the outgoing RCers.
 
-I was surprised at my feelings after sharing those things. Letting others know my feelings is important to me, and I expect others to value receiving that information.
+I was surprised by how I felt after sharing those things. Letting others know my feelings is important to me, and I expect others to value receiving that information.
 
-Some people thanked me afterwards for my niceities and that felt really good. Knowing that they valued hearing my thoughts as much as I valued sharing them.
-Other people didn't say anything. As if whatever I had to say was just another "It was nice to meet you!". That bothered me a bit.
+Some people thanked me afterwards for my niceities and that felt really good... knowing that they valued hearing my thoughts as much as I valued sharing them.
+Other people didn't acknowledge it. It was as if whatever I had to say was just another "It was nice to meet you!". That surprised me a bit.
```

Comment:
thank you for publicly acknowledging your surprise! i think a lot of people might feel the same